crates/oxc_linter/src/snapshots/eslint_require_await.snap

```diff
@@ -6,57 +6,67 @@ source: crates/oxc_linter/src/tester.rs
  1 │ async function foo() { doSomething() }
    ·                ───
    ╰────
+  help: Consider removing the 'async' keyword.
 
   ⚠ eslint(require-await): Async function has no 'await' expression.
    ╭─[require_await.tsx:1:2]
  1 │ (async function() { doSomething() })
    ·  ──────────────────────────────────
    ╰────
+  help: Consider removing the 'async' keyword.
 
   ⚠ eslint(require-await): Async function has no 'await' expression.
    ╭─[require_await.tsx:1:1]
  1 │ async () => { doSomething() }
    · ─────────────────────────────
    ╰────
+  help: Consider removing the 'async' keyword.
 
   ⚠ eslint(require-await): Async function has no 'await' expression.
    ╭─[require_await.tsx:1:1]
  1 │ async () => doSomething()
    · ─────────────────────────
    ╰────
+  help: Consider removing the 'async' keyword.
 
   ⚠ eslint(require-await): Async function has no 'await' expression.
    ╭─[require_await.tsx:1:10]
  1 │ ({ async foo() { doSomething() } })
    ·          ───
    ╰────
+  help: Consider removing the 'async' keyword.
 
   ⚠ eslint(require-await): Async function has no 'await' expression.
-   ╭─[require_await.tsx:1:20]
+   ╭─[require_await.tsx:1:17]
  1 │ class A { async foo() { doSomething() } }
-   ·                    ────────────────────
+   ·                 ───
    ╰────
+  help: Consider removing the 'async' keyword.
 
   ⚠ eslint(require-await): Async function has no 'await' expression.
-   ╭─[require_await.tsx:1:19]
+   ╭─[require_await.tsx:1:16]
  1 │ (class { async foo() { doSomething() } })
-   ·                   ────────────────────
+   ·                ───
```

Comment:
Nothing to do with the PR:
I would prefer a consistence diagnostic span. Sometimes we are including the func body, sometimes not